packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -49,12 +51,20 @@ const ExperimentMainContent = ( {
 		return <Spinner />;
 	}
 
+	const label = navigationMenu
+		? sprintf(
+				/* translators: %s: The name of a menu. */
+				__( 'Navigation menu: %s' ),
+				navigationMenu.title
+		  )
+		: __( 'No navigation menu selected, displaying Page List' );
```

Comment:
```suggestion
		: __( 'You have not yet created any menus. Displaying a list of your Pages' );
```

I wonder if this would be preferable? I don't like "Page List". It's very technical. The change is minor but I think it's clearer.

Also if there are no menus let's not suggest they can select one. Let's explain that they haven't created one yet.